rllib/algorithms/impala/tests/tf/test_vtrace_v2.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+import numpy as np
+
+from gymnasium.spaces import Box, Discrete
+
+from ray.rllib.algorithms.impala.tf.vtrace_tf_v2 import (
+    vtrace_tf2,
+    make_time_major,
+)
+from ray.rllib.algorithms.impala.tests.test_vtrace import (
+    _ground_truth_vtrace_calculation,
+)
+from ray.rllib.models.tf.tf_action_dist import Categorical
+from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.utils.test_utils import check
+
+tf1, tf, _ = try_import_tf()
+tf1.enable_eager_execution()
+
+
+def flatten_batch_and_time_dim(t):
```

Comment:
this is the same test that we merged and so im forgetting if there is a reason that it looks like we added this as a new test.

Reply:
I did some digging ... I renamed the test and moved it to somewhere else in the file tree.